### DISCUSSION TITLE
Filterable text field default behavior can be "where like contains"

### DISCUSSION THREAD

**Discussion by suleymanozev** [laravel/nova-issues#4096] [Q&A]
After all, it is a behavior that appeals to everyone. Even if it is equal or contains, it brings results. Some long text fields can be problematic.

**Comment by crynobone** [comment]
Use 

```php
Text::make('Email')->filterable(function ($request, $query, $value, $attribute) {
    return $query->where($attribute, 'LIKE', $value);
}),
```

**Reply by suleymanozev** [reply]
Thanks, I've seen this, but I think it's pointless to apply it to every field.

**Reply by suleymanozev** [reply]
What I suggest is to implement this in Laravel\Nova\Fields\Text class

**Reply by suleymanozev** [reply]
This doesn't break the existing behavior

**Reply by suleymanozev** [reply]
If you add this code to Laravel\Nova\Fields\Text class, what I want is done. Both contains and equals work. Current behavior only works as equals.

``` php
protected function defaultFilterableCallback()
{
    return function (NovaRequest $request, $query, $value) {
        return $query->where($this->filterableAttribute($request), 'LIKE', "%{$value}%");
    };
}
```

**Reply by suleymanozev** [reply]
@crynobone
